packages/examples/packages/wasm/src/index.ts

```diff
@@ -1,13 +1,14 @@
 import { rpcErrors } from '@metamask/rpc-errors';
 import type { OnRpcRequestHandler } from '@metamask/snaps-sdk';
 
+// This is only imported for its type. It is not used at runtime.
 // eslint-disable-next-line import/order
-import { instantiate } from '../build/program';
+import type { instantiate } from '../build/program';
 
-// This is the WASM module, generated by AssemblyScript, inlined as a
-// `Uint8Array` by the MetaMask Snaps CLI loader.
+// This is the WASM module, generated by AssemblyScript, inlined as an object
+// containing the functions exported by the WASM module.
 // eslint-disable-next-line import/extensions
-import program from '../build/program.wasm';
+import * as program from '../build/program.wasm';
```

Comment:
Do we need to change the import style here?

Reply:
Yeah, since the module doesn't have a default export. We use `program[method]`, and importing like this makes that possible.